apps/backend/src/routes/user/refresh_intersection.ts

```diff
@@ -64,6 +67,22 @@ router.post(
           }
         }
 
+        if (
+          hotTakes0.length !== 0 &&
+          hotTakes1.length !== 0 &&
+          hotTakes0.length === hotTakes1.length
+        ) {
+          for (let i = 0; i < hotTakes0.length; i++) {
+            const hotTake0 = hotTakes0[i];
+            const hotTake1 = hotTakes1[i];
+            if (hotTake0 === hotTake1) {
```

Comment:
Flipping this equality was the only alteration I made the tensions backend pattern, to flip from disagreements to agreements. I'm slightly more inclined towards disagreements (because it's more fun) but having both shows more versatility of PSI.